Validator/Constraints/PhoneNumberValidator.php

```diff
@@ -122,4 +124,35 @@ private function addViolation($value, Constraint $constraint)
             ->setCode(PhoneNumber::INVALID_PHONE_NUMBER_ERROR)
             ->addViolation();
     }
+
+    /**
+     * Select the region.
+     *
+     * @param Constraint $constraint The constraint for the validation.
+     *
+     * @return string Region code (2 digits)
+     *
+     * @throws ConstraintDefinitionException
+     * @throws MissingDependencyException
+     */
+    private function getRegion(Constraint $constraint)
+    {
+        $object = $this->context->getObject();
+        $path = $constraint->regionProperty;
+
+        if (null !== $path) {
+            if (!class_exists('\Symfony\Component\PropertyAccess\PropertyAccess')) {
+                throw new MissingDependencyException('You should install "symfony/property-access" in order to use the "path" attribute.');
```

Comment:
`throw new \LogicException(sprintf('The "XXX" property requires the "PropertyAccess" component. Try running "composer require symfony/property-access".'));`